src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -60,6 +60,7 @@
 import com.google.common.collect.*;
 import com.google.common.util.concurrent.*;
 
+import org.apache.cassandra.auth.*;
```

Comment:
nit: Consider restoring individual imports.

Reply:
Bah! Something in my IDE is clearly misbehaving. Sorry for these stragglers; I'll get to the bottom of it.